topology/probes/netlink.go

```diff
@@ -437,6 +460,35 @@ func (u *NetNsNetLinkProbe) addLinkToTopology(link netlink.Link) {
 	u.handleIntfIsVeth(intf, link)
 }
 
+func (u *NetNsNetLinkProbe) getRoutingTable(link netlink.Link) *RoutingTable {
+	routeList, err := u.handle.RouteList(link, netlink.FAMILY_ALL)
+	if err == nil && len(routeList) > 0 {
+		var routeTable = RoutingTable{Id: routeList[0].Table, Src: routeList[0].Src}
+		var routes []Route
+		for _, route := range routeList {
+			var r = Route{}
+			if route.Dst != nil {
+				r.Prefix = (*route.Dst).IP
+			}
+			var nh []NextHop
+			if len(route.MultiPath) > 0 {
+				for _, nexthop := range route.MultiPath {
+					var nhop = NextHop{Ip: nexthop.Gw, Priority: route.Priority}
+					nh = append(nh, nhop)
+				}
+			} else {
+				var nhop = NextHop{Ip: route.Gw, Priority: route.Priority}
+				nh = append(nh, nhop)
+			}
+			r.Nexthops = nh
+			routes = append(routes, r)
```

Comment:
I don't know why but multiple nexthops are reported in different routes having the same prefix

Reply:
if i am not getting the multipath/Nexthop i am providing the route.Gw value similarly how the netlink string() method provides. this may be the reason. if it is not required i can remove this part but in my system i am always getting multipath array is empty.